### ISSUE TITLE
Hardware acceleration support

### ISSUE THREAD

**Issue by pierreluctg** [intrepidcs/python_ics#19]
Hi @drebbe-intrepid, is it possible to use the neoVI devices hardware acceleration outside of Vehicle
Spy?

For example, would it be possible to do periodic messaging on the HW itself with `python-ics`?

**Comment by drebbe-intrepid** [comment]
@pierreluctg You would still have to use Vehicle Spy to generate the coremini and then you could use `ics.coremini_load()` and `ics.coremini_start()` to start it from python.

Its not in the best location but inside `python_ics/test/obsolete/cmvspyconvert.py` you can take the generated coremini header and embedded it inside your python script.

**Comment by pierreluctg** [comment]
@drebbe-intrepid, is it possible to create a single coremini that can be used on any neoVi devices?

**Comment by drebbe-intrepid** [comment]
@pierreluctg yes, as long as the firmware versions match with the version of Vehicle Spy you created the coremini in.
